lib/open_project/costs/patches/projects_controller_patch.rb

```diff
@@ -30,7 +30,7 @@ module InstanceMethods
     def own_total_hours
       if User.current.allowed_to?(:view_own_time_entries, @project)
         cond = @project.project_condition(Setting.display_subprojects_work_packages?)
-        @total_hours = TimeEntry.visible.sum(:hours, include: :project, conditions: cond).to_f
+        @total_hours = TimeEntry.visible.includes(:project).conditions(cond).sum(:hours).to_f
```

Comment:
Is `conditions(cond)` an actual thing?
Even if, aren't we always using `where`?